examples/secmaster/workflow-version/variables.tf

```
@@ -0,0 +1,75 @@
+# Variable definitions for authentication
+variable "region_name" {
+  description = "The region where the workspace is located"
+  type        = string
+}
+
+variable "access_key" {
+  description = "The access key of the IAM user"
+  type        = string
+  sensitive   = true
+}
+
+variable "secret_key" {
+  description = "The secret key of the IAM user"
+  type        = string
+  sensitive   = true
+}
+
+# Variable definitions for resources/data sources
+variable "workspace_id" {
+  description = "The ID of the workspace"
+  type        = string
+  default     = ""
+}
```

```suggestion
variable "workspace_id" {
  description = "The ID of the workspace"
  type        = string
  default     = ""
  nullable    = false
}
```
Adding nullable = false to ternary operator logic